saleor/graphql/core/utils/error_codes.py

```diff
@@ -0,0 +1,177 @@
+from enum import Enum
+
+DJANGO_VALIDATORS_ERROR_CODES = [
+    "invalid",
+    "invalid_extension",
+    "limit_value",
+    "max_decimal_places",
+    "max_digits",
+    "max_length",
+    "max_value",
+    "max_whole_digits",
+    "min_length",
+    "min_value",
+    "null_characters_not_allowed",
+]
+
+DJANGO_FORM_FIELDS_ERROR_CODES = [
+    "contradiction",
+    "empty",
+    "incomplete",
+    "invalid_choice",
+    "invalid_date",
+    "invalid_image",
+    "invalid_list",
+    "invalid_time",
+    "missing",
+    "overflow",
+    "required",
+]
+
+DJANGO_PASSWORD_VALIDATION_ERROR_CODES = [
+    "password_entirely_numeric",
+    "password_too_common",
+    "password_too_short",
+    "password_too_similar",
+]
+
+
+class AccountErrorCode(Enum):
```

Comment:
I think it would be better to put them to the concerned packages, i.e. `AccountErrorCode` into `graphql.account`.

Not sure where. The current convention for enums is to put them in the `__init__.py` but I'm not sure if that's the best place, especially as we don't always expect to have other stuff than imports inside this file.

Reply:
cc. @maarcingebala

Reply:
I would be glad to hear your suggestion. I do not insist with that solution. :)

Reply:
Fixed, now they sit next to models.